README.md

```diff
@@ -161,10 +161,35 @@ app.listen(3001);
 
 ### Hapi
 
+#### Version 17+
 ```js
 import hapi from 'hapi';
 import { hapi as voyagerMiddleware } from 'graphql-voyager/middleware';
 
+const server = new Hapi.Server({
+  port: 3001
+});
+
+const init = async () => {
+  await server.register({
+    plugin: voyagerMiddleware,
+    options: {
+      path: '/voyager',
+      endpointUrl: '/graphql'
+    }
+  });
+
+  await server.start();
+};
+
+init();
```

Comment:
Not a HAPI user but it took me some mental effort to understand that you need `init` just to chain two promisses.
Can it be written like that:
```js
async function startServer() {
  const server = new Hapi.Server({
    port: 3001
  });

  await server.register({
    plugin: voyagerMiddleware,
    options: {
      path: '/voyager',
      endpointUrl: '/graphql'
    }
  });
 
  await server.start();

  return server;
};
 
init();
```

Reply:
There are many different ways to write this code.  I was just using the most basic example from https://hapijs.com/tutorials as the basis

Reply:
Makes sense to use something familiar to HAPI community 👍